webapp/src/pages/Mobility.js

```diff
@@ -1,23 +1,30 @@
+import mobilityHeaderImage from "../assets/images/onboarding-mobility.jpg";
 import ExternalLink from "../components/ExternalLink";
+import WaitingListPage from "../components/WaitingListPage";
 import Map from "../components/mobilitymap/Map";
 import { t } from "../localization";
 import externalLinks from "../modules/externalLinks";
-import { LayoutContainer } from "../state/withLayout";
 import React from "react";
 
 export default function Mobility() {
   return (
     <>
-      <LayoutContainer top gutters>
```

Comment:
Make this configured with `REACT_APP_FEATURE_MOBILITY` (see `config.js`). Don't forget to pass it in as part of the build. Then we can have it enabled on staging and disabled on production.

Reply:
@rgalanakis Okay, are you referring to making configurable the waitlist component or the map component? I'm guessing the map component so people do not see that content.

Reply:
I mean, we should be able to entirely turn off mobility with a config change, so it's just a waitlist.